views/components/Sidebar/index.jsx

```diff
@@ -6,7 +6,17 @@ import SidebarFooter from './SidebarFooter'
 
 const Sidebar = ({ user, sidebarOpen, onLogout }) => {
   return (
-    <Drawer variant="permanent" open={sidebarOpen}>
+    <Drawer
+      variant="permanent"
+      open={sidebarOpen}
+      sx={{
+        flexShrink: 0,
+        boxSizing: 'border-box',
+        width: '248px',
```

Comment:
do we need to specify the height and width..? the drawer component has it already set.

Reply:
I tried to limit the use of height and width but I do think we need it here. There was overlapping content and weird layout issues with it, I think maybe because a Drawer is not typically used for a side nav and had MUI styles to overlap content.

Reply:
I don't know that we need all of these props. To adjust the sidebar to not overlay we can do 
```javascript
 PaperProps={{sx: {position: 'static'}}}
```

Then we don't have to wrap all of these components in more divs, i'm not sure why we need the boxSizing.

Reply:
👍 makes sense - https://github.com/AMP-SCZ/dpdash/pull/472/commits/8da909b1b2fc5c3b338b95ab5e5dafa937b2e2cc